utils/psh.md

```diff
@@ -1,4 +1,4 @@
-# Phoenix Shell (psh)
```

Comment:
We can leave it as utilities are printed in the following convention:
![Screenshot from 2024-03-21 16-49-02](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/84423626-e33d-4a20-ae13-c146c652ff57)